cli/scripts/dev.sh

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
```

Comment:
Ideally we'd do this with something like xtask or cargo make, but we can use bash for now

Reply:
Looks like a great dev tool by the way